.golangci.yml

```diff
@@ -18,7 +18,6 @@
 ---
 run:
   concurrency: 6
-  deadline: 5m
```

Comment:
Removed because the property `deadline` is missing in [v1.55.2](https://github.com/golangci/golangci-lint/blob/e3c2265f4939976874989e159386b3bb7dcf8e1f/.golangci.reference.yml#L8)